Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,61 @@
+## Coaching Guide
+
+_Original author: Diana Vashti_.
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments or most of an afternoon doing code reviews off “Prrr”. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coaching.
+
+## How do I know if I am coaching?
+
+Once you make it to level 3 you will become eligible to coach. If it’s your first week at level 3 you will likely coach that week.
+
+It is important that you do not vote for a goal if you are coaching so check with the coach team lead each week you are level 3 or above to determine if you are on coach-duty or not.
+
+## What does coaching entail?
+
+Check out the [Technical Coach](../Game_Manual/Levels_and_Roles.md#technical-coach) outline in [Levels and Roles](../Game_Manual/Levels_and_Roles.md).
+
+### Okay, but what does my day actually look like?
```

Comment:
Add a responsibility perhaps, something along the lines of:
```md
As a coach you have 4 main responsibilities:

1. **Hold daily stand up meetings for your teams at 9:00am**.  These meetings should consist of every learner relating:
    1. What they did yesterday
    1. What they have planned for today
    1. What is blocking them from making progress
If a learner is blocked, you should request that they add a request to coach-queue, so that you can begin the day helping unblock your teams.
```